spec/features/collections_search.rb

```diff
@@ -0,0 +1,57 @@
+require 'spec_helper'
+
+feature 'Visitor wants to browse by unit collections' do
+  describe 'RDCP browse by collection page' do
+    before do
+      @unit = DamsUnit.create(pid: 'xx48484848', name: "Research Data Curation Program", description: "Research data", code: "rdcp", uri: "http://library.ucsd.edu/rdcp")
+      @oldest_collection = DamsProvenanceCollection.create(pid: 'collection_1', titleValue: "A Old Collection", unitURI: @unit.pid, visibility: "public", resource_type: "text")
+      @oldest_collection.damsMetadata.content = File.new('spec/fixtures/damsProvenanceCollection3.rdf.xml').read
+      solr_index @unit.pid
+      solr_index @oldest_collection.pid
+
+      Timecop.freeze(Date.today + 5) do
+        @newest_collection = DamsProvenanceCollection.create(pid:'collection_2', titleValue: "The New Collection", unitURI: @unit.pid, visibility: "public", resource_type: "text")
+        @newest_collection.damsMetadata.content = File.new('spec/fixtures/damsProvenanceCollection3.rdf.xml').read
+        solr_index @newest_collection.pid
+      end
+    end
+    after do
+      @unit.delete
+      @oldest_collection.delete
+      @newest_collection.delete
+    end
+    it 'should sort by most recent collection, descending to oldest' do
+      visit dams_unit_collections_path('rdcp')
+      old_collection_sort_order = page.body.index('A Old Collection')
+      new_collection_sort_order = page.body.index('The New Collection')
+      expect(old_collection_sort_order).to be > new_collection_sort_order
+    end
```

Comment:
Could we add a test for user selected sorting rules like` sort by collection title` to see whether it works? It seems like there could be a gap but I may be wrong. See my other comment above.

Reply:
The default sort I think is tested in the DLDP browse test I wrote below the RDCP one, which basically demonstrates the normal alphabetical sort. Can you take a look at that again and let me know if you think there's still a gap that needs testing? It's certainly possible :)

Reply:
@lsitu - Oh, I apologize. You're saying if a user selects different sorting rules after the page has initially loaded. That's a really great point, and possibly a regression. The current code might always force this sorting for RDCP. I understand what you're saying now, thanks for pointing this out. I'll think about how to adjust for this.